server/routes/viewModels/taskLists/tasks/riskManagement.js

```diff
@@ -29,8 +42,8 @@ const getLabel = ({ decisions, tasks }) => {
       return 'WARNING||Mandatory address checks not completed'
     }
 
-    if (awaitingRiskInformation) {
-      return 'WARNING||Still waiting for information'
+    if (warningLabel) {
+      return `WARNING||${warningLabel}`
```

Comment:
allows showing multiple warnings which is behaviour required if user selects `Yes` still awaiting other information and `No` to has the COM consulted the POM.